contracts/core/EntryPoint.sol

```diff
@@ -628,11 +641,6 @@ contract EntryPoint is IEntryPoint, StakeManager, NonceManager, ReentrancyGuard,
             );
         }
         unchecked {
-            uint256 gasUsed = preGas - gasleft();
-
-            if (mUserOp.verificationGasLimit + mUserOp.paymasterVerificationGasLimit < gasUsed) {
-                revert FailedOp(opIndex, "AA40 over verificationGasLimit");
```

Comment:
We removed this since we now check each of the limits separately.  Is it guaranteed that this can no longer happen even with EntryPoint overhead?  I assume it is, just checking.

Reply:
you're right. we didn't cover some validations. Now account validation gas limit would cover also nonce check and copy